src/main/java/nl/novi/techiteasy/service/TelevisionService.java

```diff
@@ -0,0 +1,158 @@
+package nl.novi.techiteasy.service;
+
+import nl.novi.techiteasy.dtos.TelevisionDto;
+import nl.novi.techiteasy.dtos.TelevisionInputDto;
+import nl.novi.techiteasy.dtos.TelevisionOutputDto;
+import nl.novi.techiteasy.exceptions.RecordNotFoundException;
+import nl.novi.techiteasy.models.Television;
+import nl.novi.techiteasy.repositories.TelevisionRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class TelevisionService {
+
+private final TelevisionRepository repos;
+
+    public TelevisionService(TelevisionRepository repos) {
+        this.repos = repos;
+    }
+
+    // onderstaande haalt data(television) uit de database models Television via de service naar de gebruiker TelevisionDto
+    // List<TelevisionDto> wordt gebruikt om geconverteerde DTO-objecten op te slaan voordat ze geroutineerd worden
+    public List<TelevisionDto> getAllTelevision (){
```

Comment:
hier bijvoorbeeld kan je de TelevisionOutPut to gebruiken in plaats of TelevisionDto.